cmd/azure.go

```diff
@@ -34,15 +34,22 @@ var (
 	azureKeyVaultName   string
 	azureKeyVaultClient keyvault.BaseClient
 	azureSecretCache    map[string]map[string]interface{}
+	azureUseAzCliLogin  = false
 )
 
 func getKeyVaultClient() keyvault.BaseClient {
-	if !initialized {
+	if !initialized && !azureUseAzCliLogin {
```

Comment:
nitpick - use 2 conditionals so it reads cleaner, and is more extensible

```
if !initialized {
  if azureUseAzCliLogin {
    // azure cli case
  } else {
    // default case
  }
}
```

Reply:
Fixed